Libraries/Image/resolveAssetSource.js

```diff
@@ -104,9 +110,13 @@ function getBasePath(asset) {
  */
 function getScaledAssetPath(asset) {
```

Comment:
It seems like you're generalizing this to work for iOS or Android, whereas this was only iOS earlier.

On Android though, you actually want this to return a `/` separated filename that's in the same folder structure as the JS code, right? Why are you using `getAssetDestPathAndroid(..)` at all?